### DISCUSSION TITLE
Categorizing and labelling discussions

### DISCUSSION THREAD

**Discussion by pawamoy** [mkdocs/mkdocs#3666] [Meta]
The default categories can probably be improved to better reflect the topics we want to discuss here.
Lets discuss which categories we should have or not have to maintain our focus where it matters most :slightly_smiling_face: 

Initial list:

- maintainership
- team organization
- issues and PRs management
- collab on specific ideas/projects

The goal is also to make it easier to *search* through discussions. The better they are categorized, the easier.

**Comment by byrnereese** [comment]
I generally agree. I recommend starting very simple and very broad. Then observe how conversations evolve, and adapt our organization to meet how people are conversing naturally. For example:

* **General**. Where most conversations can go. 
* **Maintainers**. Where everyone actively maintaining the source code can collaboration and coordinate.
* **Ideas**. Where we can discuss the roadmap and new ideas. 

Conversations about specific PRs/issues/ideas (once they are agreed upon) should be public and associated with an issue under the Github Issues tab.

**Reply by byrnereese** [reply]
One quick thing we could do is remove the following categories:
* Show and tell
* Q&A
* Polls

**Reply by pawamoy** [reply]
Good idea. Let me see if I can remove these categories.

**Reply by Andre601** [reply]
Not sure, but are sub-categories available? GitHub has them in the community repository, but I'm unsure if that is something GA or yet only available for them (or in alpha/beta).

If it is available, then perhaps we could make sub-categories for more refined ordering.

No matter the above, I think there should be categories to split discussions up for code and documentation-related conversations. Can make it easier to find them if discussions here ever reach high numbers.

**Reply by pawamoy** [reply]
Seems like it's possible to have sub-categories, yes ~~(they're called sections apparently)~~: 

![newsection](https://github.com/mkdocs/mkdocs-team-space/assets/3999221/1915cfc0-7e12-46c4-b536-76aaa3d3a390)

Looks like a section is a group of categories.

**Reply by pawamoy** [reply]
I'll move forward and apply @byrnereese's suggestion for categories.

**Reply by pawamoy** [reply]
Other platforms like the Python Discuss forum and Stack Overflow seem to have success with a "Meta" category so I'll add that one too.

**Comment by pawamoy** [comment]
What do you think about using labels? Should we define priority labels, as high, medium and low? Who is to decide the priority of something? Should we have meta-discussions with polls?

**Reply by pawamoy** [reply]
Ah, just found again a comment from @Andre601:

> Maybe add labels, so that discussions can be categorized further, i.e. with current "states" (In Progress, Planned, Paused, etc) and importance levels (Low, Medium, High).

I'll proceed with those in a few days if no one comments :slightly_smiling_face:

**Reply by pawamoy** [reply]
Done!

**Reply by pawamoy** [reply]
I've removed the status labels because we can use the [project boards](https://github.com/orgs/mkdocs/projects/1) to do that. It will also help keeping things simple here. We can revise later if needed.
